cleanup_utils/dns_cleanup.py

```diff
@@ -79,6 +86,37 @@ def get_dns_record_list(self, zone_id):
 
         return dns_entries
 
+    def delete_dns_record(self, zone_id, zone_version_id, record_name):
+        """
+        Delete a record from a version of the domain
+        """
+        # Delete record
+        try:
+            self.client_zone.record.delete(self.api_key, zone_id, zone_version_id, {
+                'type': ['A', 'CNAME'],
+                'name': record_name,
+            })
+        except xmlrpc.client.Fault as e:
+            logger.error("  > FAILED Deleting DNS entries for %s...", record_name)
+            logger.error("  > ERROR: %s", e)
+
+    def create_new_zone_version(self, zone_id):
+        """
+        Create a new version of the domain, based on the current version
+        Returns the `version_id` of the version
+        """
+        return self.client_zone.version.new(self.api_key, zone_id)
+
+    def set_zone_version(self, zone_id, zone_version_id):
```

Comment:
@giovannicimolin, similar concerns to the comment on creating a new zone version.

Reply:
Same as the comment above. 
We either move this away from the CI or make this an atomic transaction.

Reply:
@lgp171188